src/app/components/partials/header/header.component.html

```diff
@@ -15,7 +15,7 @@
         
           <div class="page__header__column page__header__column--title">
               <a class="govuk-header__link govuk-header__link--service-name ">
-                {{ userName === null ? 'Username' : userName }}
+                {{ true ? 'Username' : userName }}
```

Comment:
is this set to true for testing purposes?

Reply:
I've changed this